dispatcher/stateful.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = function StatefulDispatcher(dispatcher, data, onData) {
```

Comment:
That’s the obvious choice, of course, but the base dispatcher is stateless, too. What makes the static dispatcher special is that it will only call the callback passed to `dispatch` once.